test/verify/check-networking-firewall

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/python2
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import parent
+from testlib import *
+import time
+
+
+def wait_unit_state(machine, unit, state, timeout=10):
+    start = time.time()
+    while machine.execute("systemctl is-active {} || true".format(unit)).strip() != state:
+        time.sleep(0.1)
+        if time.time() - start > timeout:
+            raise Exception("timed out waiting for {} to become {}".format(unit, state))
+
+
+class TestNetworking(MachineCase):
```

Comment:
Please rename to `TestFirewall`, to avoid confusion.

These tests need to be blacklisted on at least rhel-7-4 (where networking is in base, and thus don't get this feature). There are also some images (like the Debians) which don't have firewalld, I suggest the test skips them for now.